minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloConfigImpl.java

```diff
@@ -164,10 +164,22 @@ MiniAccumuloConfigImpl initialize() {
       mergePropWithRandomPort(Property.MONITOR_PORT.getKey());
       mergePropWithRandomPort(Property.GC_PORT.getKey());
 
+      mergeProp(Property.COMPACTOR_PORTSEARCH.getKey(), "true");
+
+      mergeProp(Property.TSERV_COMPACTION_SERVICE_ROOT_PLANNER.getKey(),
+          Property.TSERV_COMPACTION_SERVICE_ROOT_PLANNER.getDefaultValue());
+      mergeProp(Property.TSERV_COMPACTION_SERVICE_ROOT_EXECUTORS.getKey(),
+          Property.TSERV_COMPACTION_SERVICE_ROOT_EXECUTORS.getDefaultValue());
+
+      mergeProp(Property.TSERV_COMPACTION_SERVICE_META_PLANNER.getKey(),
+          Property.TSERV_COMPACTION_SERVICE_META_PLANNER.getDefaultValue());
+      mergeProp(Property.TSERV_COMPACTION_SERVICE_META_EXECUTORS.getKey(),
+          Property.TSERV_COMPACTION_SERVICE_META_EXECUTORS.getDefaultValue());
+
       mergeProp(Property.TSERV_COMPACTION_SERVICE_DEFAULT_PLANNER.getKey(),
           Property.TSERV_COMPACTION_SERVICE_DEFAULT_PLANNER.getDefaultValue());
-      mergeProp("tserver.compaction.major.service.default.planner.opts.executors",
-          "[{\"name\":\"all\",\"type\":\"external\",\"queue\":\"defaultQueue\"}]");
+      mergeProp(Property.TSERV_COMPACTION_SERVICE_DEFAULT_EXECUTORS.getKey(),
```

Comment:
This caused me to look at Property. I didn't realize that you changed the default. That's good.

Reply:
FYI that the new defaults of user-small and user-large cause a problem in the shell scripts. I had to change the hyphens to underscores in a branch that will become a PR soon (https://github.com/dlmarion/accumulo/commit/b88efed80f8672d40d723dcd50232c5c09e66024)

Reply:
I opened #3546 about deciding what the default groups of compactors should be, I think it something we should circle back to later in the elasticity cycle.